apps/actions-dependencies-updater/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "lib": ["es2023"],
+    "module": "node16",
```

Comment:
We're using node20 in `.tool-versions`

Reply:
https://www.npmjs.com/package/@tsconfig/node20

Reply:
https://www.typescriptlang.org/docs/handbook/modules/theory.html#the-module-output-format